app/konnect/org-management/deactivation.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Account and Organization Deactivation
+no_version: true
+---
+
+A {{site.konnect_saas}} organization is automatically deactivated after 30
+days of inactivity.
+
+Your organization is considered inactive when:
```

Comment:
we de-activate account only if its free account. If its Plus or enterprise, we do not de-activate account due to in-activity.

Reply:
Does this point still apply?
> There are no API requests in either the current or the previous billing cycle.

Ie, does an Enterprise or Plus account get deactivated in this situation? If it doesn't get deactivated, is there some interim state, eg dormant, where the user is not billed?

Reply:
As far as I know, weekly task that disables organizations do not modify any orgs which belong to Plus or enterprise tier.

Reply:
So then the point "There are no API requests in either the current or the previous billing cycle" does not apply at all? The term "billing cycle" is confusing if we only use it to refer to free accounts.

Reply:
This is still one of the criteria to determine inactivity.